### DISCUSSION TITLE
大厂用kotlin吗？

### DISCUSSION THREAD

**Discussion by zand951** [Snailclimb/JavaGuide#1903] [General]
hi hi， 问一问大厂现在Java岗有转kotlin吗？若能熟练使用话是不是加分项？ QS前100会被卡学历吗？（本科）

**Comment by Snailclimb** [comment]
> hi hi， 问一问大厂现在Java岗有转kotlin吗？若能熟练使用话是不是加分项？ QS前100会被卡学历吗？（本科）

Kotlin 是一门不错的编程语言，但目前后端开发领域还是 Java 使用的比较多，未来我预测依旧如此。如果能熟练使用 Kotlin 的话，在投递需要 Kotlin 相关经验的岗位的时候肯定是加分项。

个人认为程序员招聘虽然也会看学历，但相比于其他求职方向来说还是好太多太多了。
